src/param.c

```diff
@@ -2169,42 +2169,20 @@ void VariablesInterconnect(void)
 	ipr_required=(IterMethod==IT_BICGSTAB || IterMethod==IT_CGNR);
 
 	
-	if (isUseRect) {
+	if (rectDip) {
 		maxRectScale=MAX(rectScaleX,rectScaleY);
 		MAXIMIZE(maxRectScale,rectScaleZ);
 		if (PolRelation!=POL_LDR && PolRelation!=POL_CLDR && PolRelation!=POL_CM && PolRelation!=POL_IGT_SO ) {
```

Comment:
Concerning the supported polarizabilities for rectangular dipoles. It seems that currently behavior of LDR and CLDR is identical. I propose to keep CLDR as the supported one (based on our paper) and make LDR incompatible. As I remember, current behavior corresponds more to CLDR. This will also slightly affects the logic in calculator.c. 
And then we need to make CLDR the default one for rectangular dipoles (instead of the LDR for the rest of the code) - not sure if this will affect the tests with -rect_dip 1 1 1. Let's discuss.